sdk/include/asm/asm.inc

```diff
@@ -98,7 +98,7 @@ ENDM
 ENDM
 
 .code32 MACRO
-    .code
+    .text SEGMENT use32 PUBLIC 'CODE'
```

Comment:
That's interesting. Why the usual `.code` section does not work is a mystery :thinking:

Reply:
interesting...